lib/translator/status.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_TRANSLATOR_STATUS_H
+#define LIB_TRANSLATOR_STATUS_H
+
+#include "common.h"
```

Comment:
nit: order goes like this.
Other libraries' .h files.
Your project's .h files.

should third_party be "other libraries' h files" and go first?

Reply:
I'm not too sure what counts as "other libraries" bc the scsi_defs is technically in its own library. However, when I try to rearrange the imports, clang-format changes them to be in alphabetical order

Reply:
Separate different categories of headers with an empty line. This is more readable for large groups of headers and will make clang-format behave.

Reply:
Hmm, oddly enough even when I put `common.h` on a newline, it puts it back to the same spot.
![image](https://user-images.githubusercontent.com/26905438/86158439-c50db900-bad6-11ea-8347-c1f14684c813.png)

Reply:
Ah, it looks like we have a clang-format configuration issue. The Google style default assumes `IncludeCategories` has been configured suitably to distinguish the groups automatically, but we haven't done that. We should extend out `.clang-format` with either `IncludeBlocks: Preserve` or by defining an include category for `third_party`.

Reply:
Perfectly fine to consider that out of scope for this PR, though, IMO.

Reply:
https://github.com/googleinterns/scsi2nvme/issues/94 created an issue to address this